tests/pyexample/example/enum_type.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class EnumFoo(Enum):
+    """
+    Bases: enum.Enum.
```

Comment:
Temporarily it is required. Sphinx will not generate this. Since sphinx doesn't support enum type well, we can not detect enum type from the xml data it return. Maybe in the future we can solve this.